seqr/views/apis/anvil_workspace_api.py

```diff
@@ -110,6 +127,17 @@ def create_project_from_workspace(request, namespace, name):
     if errors:
         return create_json_response({'errors': errors}, status=400)
 
+    # Validate the VCF to see if it contains all the required samples
+    samples = get_vcf_samples(data_path)
+    if not samples:
+        return create_json_response(
+            {'error': 'Bad VCF file or no sample found in the data file.'}, status=400)
```

Comment:
I don't like using the word "Bad" here. I would just say "No samples found in the provided VCF. This may be due to a malformed file"